src/main/java/org/folio/rest/impl/InventoryHelper.java

```diff
@@ -303,6 +303,7 @@ private String buildLookupEndpoint(String type, Object... params) {
                 .map(itemId -> new Piece().withFormat(pieceFormat)
                                           .withItemId(itemId)
                                           .withPoLineId(polId)
+                                          .withReceiptDate(compPOL.getReceiptDate())
```

Comment:
the receipt data on compPOL set after the line is entirely received, it'll always be null here
https://github.com/folio-org/mod-orders/blob/e1370f278e30ad4d4d52be7c7b8f9e785d6adae9/src/main/java/org/folio/orders/utils/HelperUtils.java#L1117

Reply:
Hmm. Do you know where the value comes from?

Reply:
As I understand it has to come from the UI while checkin 
![Screen Shot 2020-01-09 at 4 34 56 PM](https://user-images.githubusercontent.com/41596468/72106812-14c8b000-32fe-11ea-91b7-bf9e636d11a5.png)

Reply:
receipt_date is a mandatory field only while creating pieces using checkin, as pieces for receiving don't come from the UI, 
1) either populate it with some arbitrary date 
2) that date cannot be mandatory on schema, we just have a check on BL